test/web3c.js

```diff
@@ -144,36 +133,4 @@ describe('Web3', () => {
     // since the client uses a different ephemeral key each time, it
     // won't always be able to decode the returned log.
   }).timeout(timeout);
-
-  it('should estimate gas for confidential transactions the same as gas actually used', async () => {
```

Comment:
why can't this one be used?

Reply:
(we also want to make sure that the client library internally runs estimate gas and applies it correctly)

Reply:
Agreed we want this. However, we should change the test. 

It shouldn't test that the gas is the same. It should just test that estimateGas returns something. The "sameness" point of this test is really runtime-specific behavior.

Reply:
Updated the test to reflect the client behavior we care about.